### ISSUE TITLE
manifest for fluxweb/backend:v0.3.x not found

### ISSUE THREAD

**Issue by jbuettnerbild** [flux-web/flux-web#127]
please build and push a docker image for v0.3.x

THX

**Comment by stevenpall** [comment]
attn: @derkinderfietsen

These recent tags don't seem to have built corresponding images. This has broken our deployments, so if we could get an updated image pushed it would be much appreciated.

**Comment by jbuettnerbild** [comment]
any progress?

**Comment by jbuettnerbild** [comment]
image still doesn't exist

**Comment by stevenpall** [comment]
> image still doesn't exist

Once #135 is merged into master and a new tag is created this should be resolved.

**Comment by jbuettnerbild** [comment]
in the default values of the helm chart the tag v3.0.3 is referenced. see https://github.com/flux-web/flux-web/blob/d63ed3355ac40b7a07aa41f02887edf501802017/chart/flux-web/values.yaml#L12
Is it possible to build all v3.x.x versions afterwards.

**Comment by jbuettnerbild** [comment]
> > image still doesn't exist
> 
> Once #135 is merged into master and a new tag is created this should be resolved.

I've wanted to try the new version for months, but nothing happens here. That's sad.
